### ISSUE TITLE
ide without camstyle got higher mAP(my batch-size is 64)

### ISSUE THREAD

**Issue by jian-xu** [zhunzhong07/CamStyle#4]
Sorry to interrupt again, when I ran "python main.py -d market --logs-dir logs/market-ide" I got this
![2018-05-21 12-36-53](https://user-images.githubusercontent.com/38688364/40297662-1d23f240-5d13-11e8-8c05-94b46ef1bcc8.png)
so I just comment the camstyle part in main.py(sepecficly, the function named get_data and the function named main), then I got the result like this
mAP : 70.2%
top-1 : 87.9%
then I got everything back, I ran "python main.py -d market --logs-dir logs/market-ide-camstyle --camstyle 22", I suppose the number behind --camstyle divided by batch-size is the ratio between fake and real images.
However I got the mAP of 68.6% and top-1 of 87.5%, so the result was deteriorating after I added the camstyle image. I hope you can solve my problem. Thank you.

**Comment by zhunzhong07** [comment]
I fixed the bug of camstyle_loader, could you try to run it again? In my experiment, the rank-1 accuracy is around about 85% without CamStyle.

**Comment by jian-xu** [comment]
OK, I am running now. And I want to ask the meaning of the number given by arg.camstyle. I would like to try some different ratios between real and fake images.

**Comment by jian-xu** [comment]
Here is my understanding about the number given by args.camstyle: every time I choose 128 images from trainset, I also take 46 images from camstyle, then train the network (using different loss function).

**Comment by zhunzhong07** [comment]
Yes, the overall input images are 128+46. The ratio of real and fake is 128/46.
